src/test/ui/const-generics/integer-literal-generic-arg-in-where-clause.rs

```diff
@@ -0,0 +1,14 @@
+// check-pass
+
+#![feature(const_generics)]
+//~^ WARN the feature `const_generics` is incomplete and may cause the compiler to crash
+
+fn takes_closure_of_array_3<F>(f: F) where F: Fn([i32; 3]) {
```

Comment:
Can you extend this regression test to APIT and RPIT as well?

Reply:
Something like that?

Reply:
Yep, thanks.